lib/options/sender_option_parser.rb

```diff
@@ -117,19 +127,32 @@ def initialize(args)
         @options.msg_content_type = msg_content_type
       end
 
+      # Content type
+      @opt_parser.on(
+        "--content-type TYPE",
+        %w(string int long float bool),
+        "cast --msg-content, -list-item, and =values of -map-item" +
+        "and --msg-property to this type, default: #{Defaults::DEFAULT_CONTENT_TYPE}"
+      ) do |content_type|
+        @options.content_type = content_type
+      end
+
       # Message property
       @opt_parser.on(
         "-P",
         "--msg-property KEY=VALUE",
         String,
         "property specified as KEY=VALUE (use '~' instead of '=' for auto-casting)"
       ) do |msg_property|
-        if @options.msg_properties.nil?
-          @options.msg_properties = {}
-        end
+        _ = parse_kv(msg_property)  # ensure correct option format
```

Comment:
I think that we do not need to assign result of parse_kv to underscore, only call function.

Reply:
`_ =` suppresses the warning about unused return value

Reply:
ok